Makefile

```diff
@@ -78,16 +78,26 @@ down: ## remove stack (warning: it removes the database container)
 	@$(COMPOSE) down || echo WARNING: unable to remove the stack. Try to stop linked containers or networks first.
 .PHONY: down
 
-fixtures: \
-    run
-fixtures: ## Load test data (for development)
-	@echo "Wait for databases to be up..."
+fixtures-es:
+	@echo "Wait for elasticsearch to be up..."
 	@$(WAIT_ES)
-	@$(WAIT_MYSQL)
 	zcat ./fixtures/elasticsearch/lrs.json.gz | \
 	  $(COMPOSE_RUN) patch_statements_date | \
-	  $(COMPOSE_RUN) ralph push -b es --es-index statements-fixtures  && \
-	  scripts/users-permissions.sh
+	  $(COMPOSE_RUN) ralph push -b es --es-index statements-fixtures
+fixtures-es: ## Load LRS elasticsearch test data
+.PHONY: fixtures-es
+
+fixtures-mysql: ## Load edxapp mysql test data
+	@echo "Wait for mysql to be up..."
+	@$(WAIT_MYSQL)
+.PHONY: fixtures-mysql
+
+fixtures: \
+	run \
+	fixtures-es \
+	fixtures-mysql
+	eval 'scripts/users-permissions.sh grafana'
+fixtures: ## Load test data (for development)
```

Comment:
Is this required for this feature? Also I'm not sure to understand why you need to `eval` here (and there is a block indentation issue).

Reply:
Not necessary anymore